CONTRIBUTE.md

```diff
@@ -1,84 +1,93 @@
 # Contributing to this Repository
 
+> Please submit a pull request early in the development phase, outlining the changes you intend to make or features you intend to add. This allows us to offer feedback early on, ensuring your contribution can be added to the repository before you invest a significant amount of time.
```

Comment:
@ebolch, do we want them to submit a PR or should that information be added to an Issue?

Reply:
I think having a contributor submit a PR will make it easier to review